test_files/export/export.tsickle.ts

```diff
@@ -16,4 +19,4 @@ let /** @type {number} */ export2 = 3;
 
 // This is just an import, so export5 should still be included.
 import {export5} from './export_helper';
-const export5: NeverTypeCheckMe = export5;  /* local alias for Closure JSDoc */
+const tsickle_module_prefix_4 = goog.require('test_files.export.export_helper');
```

Comment:
Actually, fixed.

It turns out if we `goog.require` everything, even if we're just using types from it, we cause AJD to re-order our sources in a way that changes loading semantics. I've sent an email on this, I think it's generally interesting.

The workaround is to just use `goog.forwardDeclare` for everything (this is the intended use case for it).